### DISCUSSION TITLE
writng to s3 store

### DISCUSSION THREAD

**Discussion by fairyeszhuo** [pangeo-data/escience-2022#17] [Q&A] [answered]
I tried the s3 store, but can't retrieve S3 credentials: at https://dashboard.cloud.muni.cz/auth/login/?next=/
It always shows: Login failed: You are not authorized for any projects or domains.
We have already joined the pangeo project or?
Any of you also tried?

**Comment by MichaelSchulzMETNO** [comment, marked as answer]
You need to create and store the credentials following the instructions::

https://pangeo-data.github.io/escience-2022/examples/EOSC_to_bucket.html

you need to store them in your home under as explained above
.aws/credentials file

And then a working example of writing to the bucket is this one:
https://pangeo-data.github.io/escience-2022/examples/notebooks/save_to_cesnet_bucket_netcdf.html

or this one in the repo clone
escience-2022/tutorial/examples/notebooks/save_to_cesnet_bucket_netcdf.ipynb
